### DISCUSSION TITLE
Jamulus client 3.8.1, fader level moving randomly

### DISCUSSION THREAD

**Discussion by bhr-private** [jamulussoftware/jamulus#2160] [Technical Support & Help]
Hello, I have a weird symptom with v.3.8.1 (think it started with that version) on my Mac (Monterey) and regardless of enabled/disabled screen recording settings security&Privacy).

I do play, all of sudden all fader levers move to 0% although I still can continue, hear the other musicians. I then move the levels again to my preferred settings - but as I assume whenever someone is entering or leaving the session, faders go down again; same for the reverb.

Also, the setting "set all faders to new client level" does not work, faders do not move to the new client level (65%).

I can't remember having had these symptoms with former client versions - so this is either an OS issue (macOS Monterey) respectively the combination of OS version and Jamulus client version.

Anyone having the same issues?

BR
bernhard (timton)

**Comment by ann0see** [comment]
Seems like a bug, so moving it to the issues tab such that more developers see it

**Comment by ann0see** [comment]
Please see #2164 for further discussion
